c3po/db/metadata.py

```diff
@@ -9,6 +9,7 @@
 from c3po.db.models import (Artist, ArtistGenre, ArtistSong, Genre, Link, Song,
                             SongGenre, User, UserLikes, UserPosts)
 
+import logging
```

Comment:
Can you quickly update the logs to use [module-level loggers](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial) (so we can actually see that the logs are coming from `metadata.py`?

Doing so would be to say:
```py
logger = logging.getLogger(__name__)
```
and use `logger.info` to print logs.

Reply:
Gotcha